### PR TITLE
Fixes issue #127

### PR DESCRIPTION
**Pull request by fulv**

### CODE REVIEW

File: roles/varnish/templates/default.vcl.j2 [hunk 1]
```diff
@@ -74,7 +74,7 @@ sub vcl_recv {
 {%if proxy_cache_block_basic_auth|default(True) %}
     # Don't allow basic auth via this route.
     # If you want to login at the Zope root, use an ssh tunnel.
-    if (req.http.Authorization) {
+    if (req.http.Authorization && req.http.Authorization !~ 'Bearer') {
```
Comment (stevepiercy) [inline roles/varnish/templates/default.vcl.j2]: What is the `!~` operator? That's a new one to me.

Comment (fulv) [reply roles/varnish/templates/default.vcl.j2]: https://varnish-cache.org/docs/2.1/tutorial/vcl.html#operators

`~` is a regex match and `!~` negates it, so it means "does not match".

Comment (fulv) [reply roles/varnish/templates/default.vcl.j2]: Note that both `~` and `!~` are used elsewhere in the same vcl templates.

Comment (stevepiercy) [reply roles/varnish/templates/default.vcl.j2]: Oh, derp! I thought it was Python, not Varnish config.

**Pull request merged**